launch/ndt_localizer.launch

```diff
@@ -30,7 +30,7 @@
   <node pkg="ndt_localizer" type="ndt_localizer_node" name="ndt_localizer_node" output="screen">
 
     <remap from="points_raw" to="$(arg input_sensor_points_topic)" />
-    <remap from="initialpose" to="$(arg input_initial_pose_topic)" />
```

Comment:
A remap was correct as `initialpose` is a topic name used and not a param in ndt.cpp

Reply:
Will revert it in the next PR